puddlestuff/tagsources/discogs.py

```diff
@@ -183,44 +173,8 @@ def parse_album_json(data):
 
     return info, parse_tracklist(data['tracklist'])
 
-
-def parse_search_json(data):
-    """Parses the xml retrieved after entering a search query. Returns a
-    list of the albums found.
-    """
-
-    results = data.get('results', []) + data.get('exactresults', [])
-
-    if not results:
-        return []
-
-    albums = []
-
-    for result in results:
-        info = result.copy()
-        try:
-            artist, album = result['title'].split(' - ')
-        except ValueError:
-            album = result['title']
-            artist = ''
-
-        info = convert_dict(info, ALBUM_KEYS)
-
-        info['artist'] = artist
-        info['album'] = album
-
-        info['discogs_id'] = info['#r_id']
-
-        info['#extrainfo'] = (
-            translate('Discogs', '%s at Discogs.com') % info['album'],
-            SITE_URL + info['discogs_uri'])
-
-        albums.append(capture_dict_values(info))
-
-    return albums
-
-
-def retrieve_album(info, image=LARGEIMAGE, rls_type=None):
+  
+  def retrieve_album(info, image=LARGEIMAGE, rls_type=None):
```

Comment:
@bernd-wechner i think this should be de-dentent 2 spaces?

Reply:
Indeed it should not be indented. A bungle in the conflict resolution.

Reply:
all good, thanks for fixing it so quickly!